docs/wiki/The CI environment - Pipeline usage.md

```diff
@@ -0,0 +1,141 @@
+This section provides a guideline on how to use the CARML CI environment pipelines.
+
+---
+
+### _Navigation_
+
+
+- [Operate the module pipelines](#operate-the-module-pipelines)
+  - [Add a new module pipeline](#add-a-new-module-pipeline)
+- [Operate the dependency pipelines](#operate-the-dependency-pipeline)
+  - [Add a new dependency](#add-a-new-dependency)
+- [DevOps-Tool-specific](#devops-tool-specific-guidance)
+  - [GitHub workflows](#github-workflows)
+  - [Azure DevOps pipelines](#azure-devops-pipelines)
+
+---
+
+# Operate the module pipelines
+
+To validate any updates you did to a module template you can perform the following steps:
+
+1. (Optionally) Update the module's parameter files to reflect your changes.
+1. Push the local changes to the repository (using a branch that is not [main\|master]).
+1. On the DevOps platform, navigate to your pipelines and select the pipeline that was registered for the module you updated.
+1. Select the branch with your updated template.
+1. (Optionally) disable the `Remove deployed module` input parameter in case you don't want to apply the default behavior, and want to skip the deletion of the test-deployed resource to check it after deployment.
```

Comment:
```suggestion
1. (Optionally) disable the `Remove deployed module` input parameter in case you don't want to apply the default behavior and want to skip the deletion of the test-deployed resources to check them post-deployment.
```